src/infra/config.jl

```diff
@@ -88,3 +88,31 @@ Base.@kwdef struct ColBERTConfig
     nprobe::Int = 2
     ncandidates::Int = 8192
 end
+
```

Comment:
This looks clean; reiterating the point before, it'll be better to have a consistent format (eg. like capital letters to begin the field name).